### PR TITLE
Add the Contributor Covenant Code of Conduct v2.1

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Add the Contributor Covenant Code of Conduct v2.1.

### CODE REVIEW

**Comment by Snailedlt** [comment]
Is `info@devicon.dev` a valid email? If so, who has access to it?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+info@devicon.dev.
```
Comment (Snailedlt) [inline CODE_OF_CONDUCT.md]: Is `info@devicon.dev` a valid email? If so, who has access to it?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+Community leaders will follow these Community Impact Guidelines in determining
+the consequences for any action they deem in violation of this Code of Conduct:
+
+### 1. Correction
+
+**Community Impact**: Use of inappropriate language or other behavior deemed
+unprofessional or unwelcome in the community.
+
+**Consequence**: A private, written warning from community leaders, providing
```
Comment (Snailedlt) [inline CODE_OF_CONDUCT.md]: Is it even possible to send a private message on github?
```suggestion
**Consequence**: A written warning from community leaders, providing
```

Comment (Panquesito7) [reply CODE_OF_CONDUCT.md]: > Is it even possible to send a private message on github?

Not possible, but could probably be contacted via mail or another contact method.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Panquesito7) [reply CODE_OF_CONDUCT.md]: I believe @amacado's the only one who has access to it. 🤔

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Snailedlt) [reply CODE_OF_CONDUCT.md]: > could probably be contacted via mail or another contact method.

True, but that's only if they have their email publicly available

Comment (Panquesito7) [reply CODE_OF_CONDUCT.md]: > > could probably be contacted via mail or another contact method.
> 
> True, but that's only if they have their email publicly available

Hmm, I'm not sure if we should change this as we're taking it from [here](https://www.contributor-covenant.org/version/2/1/code_of_conduct/). 🤔
Unless we can somewhat adopt it.

Comment (Snailedlt) [reply CODE_OF_CONDUCT.md]: Yeah, thinking the same. Maybe we should let the PR stay open for a little while longer. See if anyone else butts in?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Snailedlt) [reply CODE_OF_CONDUCT.md]: damm, we gotta get access to that then

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Panquesito7) [reply CODE_OF_CONDUCT.md]: CC: @BenSouchet.

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**